src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
```

Comment:
1. it's a good practice to use object destructing
2. move conditions to variables

```suggestion
export const Person = ({ person }) => {
  const { name, age, isMarried, sex, partnerName } = person;
  const partnerStatus = sex === 'f' ? 'wife' : 'husband'
  
  return (
    <section className="Person">
```